Holovibes/includes/compute/fourier_transform.hh

```diff
@@ -113,6 +113,14 @@ class FourierTransform
      */
     void insert_moments();
 
+    /**
+     * \brief Sends the respective moment to the output display (gpu_postprocess_frame)
+     * if the corresponding image type is selected.
+     *
+     * \note This function probably only exists for test purposes.
+     */
+    void insert_moments_to_output();
```

Comment:
if it's only for debug use
```{c++}
#ifndef NDEBUG

// code

#endif 
```

but if it can be use in release mode change nothing